soh/soh/GameMenuBar.cpp

```diff
@@ -310,17 +310,17 @@ namespace GameMenuBar {
                 }
                 ImGui::EndMenu();
             }
-            
+
             UIWidgets::Spacer(0);
-            
+
```

Comment:
There's quite a lot of these formatting changes that don't really have anything to do with what's in the PR. I'd recommend reverting these changes.